boards.txt

```diff
@@ -721,6 +761,54 @@ RAK.menu.upload_method.STLink.upload.tool=stlink_upload
 3dprinter.menu.upload_method.serialMethod.upload.protocol=maple_serial
 3dprinter.menu.upload_method.serialMethod.upload.tool=serial_upload
 
+###############################
```

Comment:
Seems you forget to remove this part after moving to 3D printer board that's why there is a merge conflict. Anyway should be fine to rebase on top also.

Reply:
Conflicts resolved